server/migrations/20190311142356-create-recipe.js

```diff
@@ -0,0 +1,42 @@
+module.exports = {
```

Comment:
This is a duplicate of a migration that's already been checked in to `develop`

Reply:
Noted. I have fixed this.